### PR TITLE
Onboarding documentation

### PR DESCRIPTION
**Pull request by hanzala1234**

### CODE REVIEW

**Comment by stakater-user** [comment]
@hanzala1234 Image is available for testing. `docker pull docker-openshift-stakater-nexus.apps.prod.clusters.stakater.io/stakater/StakaterCloudDocs:SNAPSHOT-PR-116-438db674`

File: content/sre/onboarding/team-onboarding.md [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+```
+- values.yaml
+you can configure helm values as per your application requirement. We use [stakater application](https://github.com/stakater-charts/application/tree/master/application) chart as our main chart.
+```yaml
+application:
+  applicationName: <app-name>
+  space:
+    enabled: true
+  deployment:
+    image:
```
Comment (rasheedamir) [inline content/sre/onboarding/team-onboarding.md]: why these two are place holders?

File: content/sre/onboarding/team-onboarding.md [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+      repository: "DOCKER_REPOSITORY_URL"
+      tag: IMAGE_TAG
+    imagePullSecrets: "nexus-docker-config-forked"
```
Comment (rasheedamir) [inline content/sre/onboarding/team-onboarding.md]: can this be changed?

**Comment by stakater-user** [comment]
@hanzala1234 Image is available for testing. `docker pull docker-openshift-stakater-nexus.apps.prod.clusters.stakater.io/stakater/StakaterCloudDocs:SNAPSHOT-PR-116-012c562d`

File: content/sre/onboarding/team-onboarding.md (hunk 1 above)
Comment (hanzala1234) [reply content/sre/onboarding/team-onboarding.md]: because we should not hardcode the image name in documentation.

**Comment by stakater-user** [comment]
@rasheedamir Image is available for testing. `docker pull docker-openshift-stakater-nexus.apps.prod.clusters.stakater.io/stakater/StakaterCloudDocs:SNAPSHOT-PR-116-8d5c95df`

File: content/sre/onboarding/team-onboarding.md [hunk 3]
```diff
@@ -0,0 +1,152 @@
...
+    - port: 8080
+      name: http
+      targetPort: 8080
+  # Openshift Routes
+```
+### 2. GitOps-Config Repo
+
+Simply setting the webhook_url is sufficient to have Tekton build the application.
+
+To deploy, you'll need to add 3 files to the gitops repository.
```
Comment (rasheedamir) [inline content/sre/onboarding/team-onboarding.md]: can you list the names of those 3 files?

**Comment by stakater-user** [comment]
@rasheedamir Image is available for testing. `docker pull docker-openshift-stakater-nexus.apps.prod.clusters.stakater.io/stakater/StakaterCloudDocs:SNAPSHOT-PR-116-c5f3396a`

**Comment by stakater-user** [comment]
@hanzala1234 Image is available for testing. `docker pull docker-openshift-stakater-nexus.apps.prod.clusters.stakater.io/stakater/StakaterCloudDocs:SNAPSHOT-PR-116-d549ce04`

**Comment by stakater-user** [comment]
@hanzala1234 Image is available for testing. `docker pull docker-openshift-stakater-nexus.apps.prod.clusters.stakater.io/stakater/StakaterCloudDocs:SNAPSHOT-PR-116-cd004dc1`

**Comment by stakater-user** [comment]
@hanzala1234 Image is available for testing. `docker pull docker-openshift-stakater-nexus.apps.prod.clusters.stakater.io/stakater/StakaterCloudDocs:SNAPSHOT-PR-116-874f03cd`

**Comment by stakater-user** [comment]
@hanzala1234 Image is available for testing. `docker pull docker-openshift-stakater-nexus.apps.prod.clusters.stakater.io/stakater/StakaterCloudDocs:SNAPSHOT-PR-116-3ddf4a5e`

**Comment by stakater-user** [comment]
@hanzala1234 Yikes! You better fix it before anyone else finds out! [Build](https://github.com/stakater/StakaterCloudDocs/commit/ded4004ca6b0e3d8abb00da9517f9600e2502408/checks) has Failed!

File: .vuepress/config.js [hunk 4]
```diff
@@ -50,10 +50,18 @@ module.exports = {
                 children: [
                     '/content/sre/gitops/introduction.md',                    
                     '/content/sre/gitops/github.md',
-                    '/content/sre/gitops/gitlab.md',                    
+                    '/content/sre/gitops/gitlab.md',
+                    '/content/sre/'
```
Comment (rasheedamir) [inline .vuepress/config.js]: @hanzala1234 what is need of this?

**Comment by stakater-user** [comment]
@hanzala1234 Image is available for testing. `docker pull docker-openshift-stakater-nexus.apps.prod.clusters.stakater.io/stakater/StakaterCloudDocs:SNAPSHOT-PR-116-0769337e`

**Review by rasheedamir — APPROVED** [review]

**Comment by stakater-user** [comment]
@rasheedamir Image is available for testing. `docker pull docker-openshift-stakater-nexus.apps.prod.clusters.stakater.io/stakater/StakaterCloudDocs:v0.0.63`

**Pull request merged**